lib/Automation/Automation.h

```diff
@@ -0,0 +1,219 @@
+/*
+  Automation.h - A c++ library including automated sequences triggered by
+  commands or particular events (e.g. end of flow, overpressurization)
+*/
+
+
```

Comment:
I recall that we came to conclusion that we were going to keep the libraries a consistent format (classes w/ .h interface definition, and .cpp source). 
Low key I don't care that much. But thought I would remind you anway. 
(I've been doing this whenever I had reason to edit a specific file)

Reply:
Oh yeah good point - I'll update it to that format for consistency.